src/example/index.js

```diff
@@ -296,17 +294,13 @@ function drawCube(matrix = new Matrix4()) {
   gl.bindBuffer(gl.ARRAY_BUFFER, null);
 
   let loc = gl.getUniformLocation(shader, 'model');
-  let current = new Matrix4()
-    .multiply(matrix)
-    .multiply(model)
-    .multiply(modelScale);
-  gl.uniformMatrix4fv(loc, false, current.elements);
+  gl.uniformMatrix4fv(loc, false, matrix.elements);
```

Comment:
using the passed matrix instead of calculating one from global variables. Makes everything easier for the hair trailing